src/app/archive/lib/extensional.ml

```diff
@@ -116,7 +116,7 @@ module Block = struct
   module Stable = struct
     [@@@with_versioned_json]
 
-    module V2 = struct
+    module V3 = struct
```

Comment:
This (and similar changes) are not 'compatible', and as such can't be landed in the `compatible` branch.

Reply:
This is fixed. This work is in #15832